playbooks/files/rax-maas/plugins/ceph_monitoring.py

```diff
@@ -191,10 +191,7 @@ def get_mon_statistics(client=None, keyring=None, host=None,
                                   keyring=keyring,
                                   container_name=container_name,
                                   deploy_osp=deploy_osp)
-      mon = [m for m in ceph_mon_status['quorum']
-             if m['name'] == host]
-
-    mon_in = mon[0]['rank'] in ceph_status['quorum']
+    mon_in = host in ceph_status['quorum_names']
```

Comment:
Can you update this line in particular to work on both N and O ?

Reply:
Confirmed on nautilus that we can use that check, the full list shows in the status and when a mon service is shut down the hostname no longer appears in quorum_names in both N and O.  I'll also check pacific once I run some additional upgrades in the lab, but this should be okay for all currently deployed clusters.

(ceph_deploy) root@ceph01:/opt/ceph-ansible# ceph -v
ceph version 14.2.22 (ca74598065096e6fcbd8433c8779a2be0c889351) nautilus (stable)
(ceph_deploy) root@ceph01:/opt/ceph-ansible# ceph -f json-pretty -s |more

{
    "fsid": "5d5864ee-5619-46b2-9615-f07cdd4cda23",
    "health": {
        "checks": {},
        "status": "HEALTH_OK"
    },
    "election_epoch": 10,
    "quorum": [
        0,
        1,
        2
    ],
    "quorum_names": [
        "ceph01",
        "ceph02",
        "ceph03"
    ],